OpenRA.Game/Scripting/ScriptContext.cs

```diff
@@ -86,8 +86,8 @@ public abstract class ScriptGlobal : ScriptObjectWrapper
 		public ScriptGlobal(ScriptContext context)
 			: base(context)
 		{
-			// The 'this.' resolves the actual (subclass) type
-			var type = this.GetType();
+			// GetType resolves the actual (subclass) type
+			var type = GetType();
```

Comment:
Well we're already doing an unqualified call in the abstract [SupportPowerInfo](https://github.com/OpenRA/OpenRA/blob/9b6dba02a89f93879e151e0c6cbc5c8885e0e3dd/OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs#L58) class - so I suspect this is no longer an issue (if it ever was) or support powers would be quite broken.